devops/build_sdks.py

```diff
@@ -197,7 +206,7 @@ def main():
         build_ruby(args)
     if build_all or 'golang' in langs_to_build:
         build_golang(args)
-    if build_all or 'docs' in langs_to_build:
+    if 'docs' in langs_to_build:
```

Comment:
fix 2

Reply:
Docs should always be built because it ensures that the sample code and the documentation never go out of sync with each other.

Reply:
fair!  it seems like the docs builds usually fail - I think we might want a whole job/step in GitHub actions to configure doc building (install node, `DefaultDocumentation`, etc) and verify how we call `subprocess.Popen` in this file.  I'll see if I can fix that real quick otherwise I'll just try/catch here and let's get to those fixes in a new PR

Reply:
actually: do you think it makes sense to think of `build_{lang}_docs` as part of `build_{lang}` instead of 'docs' being its own separate thing?   I'd think the docs would only change as the source for that specific language changes.  also makes our lives easier, e.g. we don't need to install Node dependencies in a Golang build job

Reply:
I realized dotnet is a bit weird but my new impl makes sense to me - in each SDK's action, call `build_sdks.py` *with --language specified*.  when called that way, the script should only call `build_{lang}_docs`, `build_{lang}`, and `build_dotnet_docs`.  

this means we can prep dependencies easily in that SDK's action.  and it seems this runs reliably!